framework/wazuh/core/cluster/agents_reconnect.py

```diff
@@ -270,7 +270,8 @@ async def get_agents(current_balance) -> dict:
                 if info['agents'] > 0:
                     agent_query = WazuhDBQueryAgents(
                         count=False, filters={"status": "active", "node_name": node},
-                        limit=info['agents'], sort={"fields": ["id"], "order": "desc"}, select=["id"])
+                        limit=info['agents'], sort={"fields": ["id"], "order": "desc"}, select=["id"],
+                        query="version>Wazuh v4.3.0,version=Wazuh v4.3.0")
```

Comment:
In my test this is returning agent 000 of the master node, but it can't be reconnected so it should be added to the query:
```suggestion
                        query="(version>Wazuh v4.3.0,version=Wazuh v4.3.0);id!=000")
```

Reply:
Done